install/install.bash

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
```

Comment:
I see what you are trying to do with install.bash but a few concerns:

1. It doesn't feel very devops friendly - given the interactive format
2. It doesn't feel very easy to document - in the wiki guide it basically just says "run install.bash and follow the instructions"
3. It is quite restrictive - it assumes you only want to monitor one bucket, and want to monitor all ObjectCreated and ObjectRemoved events

I wonder if we turn this into a Python (for Boto compatibility) script which is supplied with a config file. The config file:

1. Can be fully documented on the wiki, making it easier to explain what is going on
2. Can be much more configurable than install.bash. For example, it can support an array of buckets to monitor, and can have some sensible (but editable) defaults for the events the user wants to monitor
3. We can namespace the configuration like so:

```yaml
snowplow:
  collector: xxx
s3:
  foo: bar
```

to future proof this against adding other event sources in the future.

1. Takes a config file